index.js

```diff
@@ -34,6 +35,17 @@ module.exports = class LernaChangelogGeneratorPlugin extends Plugin {
     return firstCommit;
   }
 
+  async _execLernaChangelog(nextVersion, from) {
```

Comment:
p1/5 super low but the order of args here feels very weird to me (to..from as opposed to from..to)